app/src/main/java/com/example/tasky/feature_agenda/data/local/AgendaDao.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.tasky.feature_agenda.data.local
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Query
+import androidx.room.Transaction
+import androidx.room.Upsert
+import com.example.tasky.feature_agenda.data.local.entity.AttendeeEntity
+import com.example.tasky.feature_agenda.data.local.entity.EventEntity
+import com.example.tasky.feature_agenda.data.local.entity.ReminderEntity
+import com.example.tasky.feature_agenda.data.local.entity.TaskEntity
+import com.example.tasky.feature_agenda.data.local.relations.EventWithAttendee
+
+@Dao
+interface AgendaDao {
+
+    @Upsert
+    suspend fun upsertEvent(event: EventEntity)
+
+    @Delete
+    suspend fun deleteEvent(event: EventEntity)
+
+    @Query("SELECT * FROM Event")
+    suspend fun getEvents(): List<EventEntity>
```

Comment:
You could also think about exposing a `Flow<List<EventEntity>>` here, so you get reactive changes